File: CarFleet-Project/Models/Tables/Brand.cs
namespace CarFleet_Project.Models.Tables
{
    public class Brand
    {
        public int brandId { get; set; } = 0;
        private string brandName { get; set; } = "";
        public virtual List<Vehicle> vehicles { get; set; } = new();
        public virtual List<Model> models { get; set; } = new();
    }
}

File: CarFleet-Project/ClientApp/src/models/Brand.ts
import { Model } from "./Model";
import { Vehicle } from "./Vehicle";

export class Brand
{
  brandId: number;
  brandName: string;
  vehicles: Vehicle[] = [];
  models: Model[] = [];

  constructor() {
    this.brandId = 0;
    this.brandName = "";
  }
}

